predicators/ground_truth_models/kitchen/options.py

```diff
@@ -238,13 +265,9 @@ def _PullKettle_terminal(state: State, memory: Dict,
                                  params: Array) -> bool:
             del memory, params  # unused
             _, obj, obj2 = objects
-            obj_y = state.get(obj, "y")
-            obj2_y = state.get(obj2, "y")
             if not GroundAtom(OnTop, [obj, obj2]).holds(state):
```

Comment:
if the latter is no longer needed, let's change this to just `return GroundAtom(...`